Keas.Core/Services/IEmailService.cs

```diff
@@ -298,6 +299,53 @@ public async Task SendExpiringMessage(int personId, ExpiringItemsEmailModel mode
             await _dbContext.SaveChangesAsync();
         }
 
+        private string BuildExpiringTextMessage(ExpiringItemsEmailModel expiringItems)
+        {
+            var message = new StringBuilder();
+            var count = expiringItems.KeySerials.Count() + expiringItems.Equipment.Count() + expiringItems.Workstations.Count() + expiringItems.AccessAssignments.Count();
+            var helplink = "https://peaks.ucdavis.edu/" + expiringItems.Person.Team.Slug +"/Help";
+
+            var plural = "item";
+            if (count > 1)
+            {
+                plural = "items";
+            }
+            message.AppendLine("It looks like you are viewing the ugly text version of this email, if you view it in html it will look a lot nicer.");

```

Comment:
nope not the hill i'm going to die on

Reply:
![image](https://user-images.githubusercontent.com/469010/55025848-f982f300-4fbe-11e9-9604-e1a405a82d94.png)